packages/typo3-docs-theme/assets/sass/mixins/_focus.scss

```diff
@@ -0,0 +1,4 @@
+@mixin focus() {
+    box-shadow: $focus-box-shadow;
```

Comment:
There seems to be a glitch, I see a black border top (Firefox):

![focus](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/49225e74-9bd7-4c7d-a9f0-36a5cb789c0f)

Also Chromium:
![focus2](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/a1d3cbaa-3d68-48cb-a433-39ad43f960c8)

Reply:
whatever, make a follow up